problem1.cs

```diff
@@ -0,0 +1,19 @@
+using System;
```

Comment:
Instead of prolem1.cs, please name this after the name of the problem

Reply:
@narayanagunturi i have done all the changes.